### DISCUSSION TITLE
Please, where is the documentation? (https://docs.bigcapital.ly/)

### DISCUSSION THREAD

**Discussion by rocral** [bigcapitalhq/bigcapital#191] [Q&A] [answered]
The internet site https://bigcapital.ly/ and the documentation site (https://docs.bigcapital.ly/) is still registered on Google but it is not available.
I would like to now more technical and functional details of the server and client, in order to evaluate my own investment to adapt the application to Spain / France / other accounting practices, or to add VAT management if it is not supported.

Regards 

Alex Garcia
Barcelona
Catalonia

**Comment by abouolia** [comment, marked as answer]
Thanks you for mentioning that, It was DNS issue because recently moved to Cloudflare and some configurations went wrong, now everything is online back.

**Reply by rocral** [reply]
Hi Ahmed, 

A very fast answer !!!

Thank you.

Alex
